src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
```

Comment:
[heave is 900](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)
<img width="445" alt="image" src="https://user-images.githubusercontent.com/86183007/184971643-8ca26d50-e303-4b72-ad8f-e65e129af96a.png">